### PR TITLE
Do not add space if empty string

### PR DESCRIPTION
**Pull request by alexndlm**

### CODE REVIEW

File: src/Ast/PhpDoc/PhpDocNode.php [hunk 1]
```diff
@@ -271,7 +271,13 @@ public function getTypeAliasImportTagValues(string $tagName = '@phpstan-import-t
 
 	public function __toString(): string
 	{
-		return "/**\n * " . implode("\n * ", $this->children) . "\n */";
+		$children = array_map(
+			static function ($child): string {
+				return (string) $child === '' ? '' : ' ' . $child;
```
Comment (JanTvrdik) [inline src/Ast/PhpDoc/PhpDocNode.php:276]: This will render each child twice.

File: src/Ast/PhpDoc/PhpDocNode.php [hunk 2]
```diff
@@ -271,7 +271,13 @@ public function getTypeAliasImportTagValues(string $tagName = '@phpstan-import-t
...
+			},
+			$this->children
+		);
+		return "/**\n *" . implode("\n *", $children) . "\n */";
```
Comment (JanTvrdik) [inline src/Ast/PhpDoc/PhpDocNode.php:280]:
```suggestion
		$children = array_map(
			static function (PhpDocChildNode $child): string {
				$s = (string) $child;
				return $s === '' ? '' : ' ' . $s;
			},
			$this->children
		);
		return "/**\n *" . implode("\n *", $children) . "\n */";
```

File: src/Ast/PhpDoc/PhpDocNode.php (hunk 1 above)
Comment (alexndlm) [reply src/Ast/PhpDoc/PhpDocNode.php]: I agree, thanks for the fix.

**Comment by ondrejmirtes** [comment]
Thank you both!

**Comment by alexndlm** [comment]
Thank you for making the release so quickly.

**Pull request merged**